2023/opmodes/javabotsLM1Auto.java

```diff
@@ -228,23 +272,19 @@ public void handleEvent(RobotEvent e) {
         });
     }
 
-    public void delay(int seconds)
+    public void delayAndLowerLift(int seconds)
     {
-        this.addTask(new SingleShotTimerTask(this, 1000*seconds) {
+        addTask(new SingleShotTimerTask(this, 1000*seconds) {
             @Override
-            public void handleEvent (RobotEvent e){
-                SingleShotTimerEvent event = (SingleShotTimerEvent) e;
-                switch(event.kind) {
-                    case EXPIRED:
-                        if (detectedAprilTagID == SIGNAL_LEFT){
-                            driveToSignalZone(leftPath);
-                        } else if (detectedAprilTagID == SIGNAL_MIDDLE){
-                            driveToSignalZone(middlePath);
-                        } else {
-                            driveToSignalZone(rightPath);
-                        }
-                        break;
-                }
+            public void handleEvent (RobotEvent e) {
+//                SingleShotTimerEvent event = (SingleShotTimerEvent) e;
+//                lowerLiftToFirstGroundJunction();
+//                switch(event.kind) {
+//                    case EXPIRED:
+                        lowerLiftToFirstGroundJunction();
+
+                       // break;
```

Comment:
Same here with removing comment out lines.

The rationale is that the only reason you might leave commented out lines in software is because you are experimenting with something and you don't want to have to recreate from scratch the original if the experiment doesn't work.

But for stuff like is expressed here, commenting out, instead of removing, just serves to muddy the software, and so it's best to just remove entirely.  You really shouldn't get in the habit of never removing in favor of always commenting out software.